index.js

```diff
@@ -135,13 +136,16 @@ function init (ssb, config) {
 
       state.loading.keystore.once((s) => {
         const peers = new Set()
-        keystore.group.list()
-          .map(groupId => keystore.group.listAuthors(groupId))
-          .forEach(authors => authors.forEach(author => peers.add(author)))
 
-        peers.delete(ssb.id)
-        Array.from(peers)
-          .forEach(id => ssb.replicate.request({ id, replicate: true }))
+        Promise.all(keystore.group.list()
+          .map(groupId => p(ssb.tribes.listAuthors)(groupId)))
+          .then((allGroupAuthors) => {
+            allGroupAuthors.forEach(authors => authors.forEach(author => peers.add(author)))
+
+            peers.delete(ssb.id)
+            Array.from(peers)
+              .forEach(id => ssb.replicate.request({ id, replicate: true }))
+          })
```

Comment:
Please use `pull-stream` for async iteraction. `pull-paramap` is incredibly powerful

```js
pull(
  pull.values(keystore.group.list()),
  pulllParaMap(
    (groupId, cb) => ssb.tribes.list.authors(groupId, (err, feedIds) => {
      if (err) {
        console.log("error with tribes.list.authors", err)
        return cb(null, [])
      }
      cb(null, feedIds)
    }),
    5
  ),
  pull.flatten(),
  pull.unique(),
  pull.drain(feedId => {
    if (feedId === ssb.id) return
    ssb.replicate.request({ id: feedId, replicate: true })
  })
```

Reply:
done, chose to error on listAuthors error though